Character.cpp

```diff
@@ -87,31 +91,48 @@ int Character::takeDamage(int damage)
 }
 
 
-#include <assert>
+#include "Utility.h"
+
+void Character::levelUp( int& currentValue, int initialValue )
+{
+    if( currentValue < initialValue )
+        currentValue = ceil(initialValue * 1.1);
+    else
+        currentValue = ceil(currentValue * 1.1);
+
+    initialValue = currentValue;
+}
```

Comment:
Good, but you need to pass `initialValue` by reference, or return currentValue and assign it to whatever stat you're updating.  Right now, you are not updating whatever you're passing to `initialValue` which means line 103 is not doing anything.
- [ ] fix param type.